docs/platform/api/getting-started-with-the-linode-api-v4.md

```diff
@@ -0,0 +1,358 @@
+    "pages": 1
+}
+{{< /file-excerpt >}}
+
+Choose one of the Linodes from the list and use its ID to view usage statistics:
+
+    curl -H "Authorization: Bearer $token" https://api.linode.com/v4/linode/instances/$id/stats
+
+## Create a Linode via API
+
+All of the examples so far have been basic *GET* requests. To add items (Linodes, IP addresses, domains, support tickets, volumes, etc.) to your account, you will need to use a *POST* request. This section guides you through the process of creating a new Linode through the API.
+
+### Get Configuration Parameters
+
+To create a Linode via API, pass configuration options to the API. Include the type/size, datacenter, and image you want to use. You will need the ID of each of these choices to use in your POST request.
+
+1.  Check available distributions:
+
+        curl https://api.linode.com/v4/images/
+
+    Choose one of the distributions from the resulting list and make a note of the `id` field. The examples in the following section use Debian 9, for which the id is `linode/debian9`.
+
+2. Repeat this for a list of available Linode types:
+
+        curl https://api.linode.com/v4/linode/types/
+
+    The type ID of a 1GB Nanode is `g5-standard-1`.
+
+3.  Choose a datacenter with the `/regions` endpoint:
+
+        curl https://api.linode.com/v4/regions
+
+### Create a Linode
+
+Replace the example IDs in this section with the IDs retrieved in the previous section. In addition, specify a root password and label for the new Linode:
+
+    curl -X POST https://api.linode.com/v4/linode/instances -d '{"type": "g5-standard-1", "region": "us-east-1a", "distribution": "linode/debian9", "root_pass": "root_password", "label": "prod-1"}' \
+    -H "Authorization: Bearer $TOKEN" -H "Content-type: application/json"
+
+  {{< file-excerpt "response.json" json >}}
+{
+  "id": 6985832,
+  "hypervisor": "kvm",
+  "region": "us-east-1a",
+  "updated": "2018-03-20T20:54:06",
+  "image": "linode/debian9",
+  "group": "",
+  "alerts": {"io": 10000, "network_in": 10, "network_out": 10, "transfer_quota": 80, "cpu": 90},
+  "backups": {"schedule": {"day": null, "window": null}, "enabled": false},
+  "status": "provisioning",
+  "created": "2018-03-20T20:54:06",
+  "type": "g5-standard-1",
+  "ipv6": "2600:3c03::f03c:91ff:fe41:a4af/64",
+  "label": "prod-1",
+  "ipv4": ["45.79.148.217"],
+  "specs": {"disk": 30720, "vcpus": 1, "memory": 2048, "transfer": 2000}
+}
+{{< /file-excerpt >}}
+
+### Reboot the Linode
+
+The new Linode should boot automatically after it is created. You can also boot, shutdown, and reboot through the API.
+
+To reboot the Linode, use its ID (included in the response in the previous section) in the following command:
+
+    curl -X POST https://api.linode.com/v4/linode/instances/$linode_id/reboot -H "Authorization: Bearer $token"
+
+If the reboot is successful, the response will be an empty JSON object (`{}`). You can boot and shut down Linodes using the `/boot` and `/shutdown` endpoints.
```

Comment:
> If the reboot is successful

That's not strictly true.  If we give you a 200, `{}` back, that means we've successfully _started_ the reboot - there's no promise it will eventually finish successfully (although it should in almost all cases).